libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -834,6 +862,13 @@ def _get_df(entries: list[CacheEntryFullMetadata]) -> pd.DataFrame:
 
 
 def get_cache_entries_df(dataset: str, cache_kinds: Optional[list[str]] = None) -> pd.DataFrame:
+    filters = {"dataset": dataset}
+    if cache_kinds:
+        filters["kind"] = {"$in": cache_kinds}  # type: ignore
```

Comment:
the syntax is not the same as `filters["kind__in"] = cache_kinds` ?

Reply:
No, I think kind__in is more for `mongoengine` but in this case it was needed a `pymongo` style